hadoop-common-project/hadoop-common/src/main/resources/core-default.xml

```diff
@@ -1933,6 +1933,20 @@
   </description>
 </property>
 
+<property>
+  <name>fs.s3a.ssl.channel.mode</name>
+  <value>Default_JSSE</value>
+  <description>
+    If secure connections to S3 are enabled, configures the SSL
```

Comment:
I don't like having to remember the case of all these options, especially as every other s3a config option is all lower case. these should all be case insensitive.